packages/scripts/package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/scripts",
-	"version": "28.5.0",
+	"version": "28.6.0",
```

`version` change is automated so this line should get reverted:

```suggestion
	"version": "28.5.0",
```